### ISSUE TITLE
Review cbrain .js files: .data() fetches adjust types instead of returnign strings.

### ISSUE THREAD

**Issue by prioux** [aces/cbrain#691]
For HTML like this:

```html
<sometag data-key="true">
```

Old jQuery used to return the string `"true"` when invoking `.data('key')`; now it converts to boolean `true`.

See : https://api.jquery.com/data/#data-html5

**Comment by prioux** [comment]
Quick grep output:

```
cbrain.js:    if (xhr.status === 0) return true;
cbrain.js:      if (keep_open !== "true") {
cbrain.js:          if (replace === "true") {
cbrain.js:          if (replace === "true") {
cbrain.js:      if (scroll_bottom === "false") scroll_bottom = false;
cbrain.js:      if (invert === "true") {
cbrain.js:        if (invert === "true") {
cbrain.js:      if (reset_form !== "false") {
cbrain.js:      if (event.keyCode !== 13) return true;
cbrain.js:      if (ajax_submit !== "false") {
userfiles.js:    if (typeof bare === 'undefined') bare = true;
```

**Comment by natacha-beck** [comment]
The second commit of this [PR](https://github.com/aces/cbrain/pull/690/commits) should fix the remaining bug.
The grep in userfile.js is not a bug is it the correct behavior.
I have done a second inspection of the file and I have seen nothing else.

**Comment by prioux** [comment]
Fixed by #690
